src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension/scripts/manageVsix.js

```diff
@@ -0,0 +1,99 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+const fetch = require('node-fetch');
+const stream = require('stream');
+const extract = require('extract-zip');
+const fs = require('fs');
+const util = require('util');
+const path = require('path');
+const { spawn } = require("child_process");
+const os = require('os');
+
+var finished = util.promisify(stream.finished);
+
+const log = (text) => console.log(`[${new Date()}] ${text}`)
+
+async function downloadProxyPackage(version) {
+    var nugetUrl = 'https://api.nuget.org/v3-flatcontainer';
+    var packageName = 'Microsoft.AspNetCore.Components.WebAssembly.DevServer';
+
+    const extractTarget = path.join(os.tmpdir(), 'blazorwasm-companion-tmp', 'extracted', `${packageName}.${version}`);
+
+    const versionedPackageName = `${packageName}.${version}.nupkg`;
+    const downloadUrl = `${nugetUrl}/${packageName}/${version}/${versionedPackageName}`;
+    const downloadPath = path.join(os.tmpdir(), 'blazorwasm-companion-tmp', versionedPackageName);
+
+    // Download and save nupkg to disk
+    log(`Fetching package from ${downloadUrl}...`)
+    const response = await fetch(downloadUrl)
+    const outputStream = fs.createWriteStream(downloadPath);
+    response.body.pipe(outputStream);
+
+    // Extract nupkg to extraction directory
+    log(`Extracting NuGet package with directory...`)
+    await finished(outputStream);
+    await extract(downloadPath, { dir: extractTarget });
+    return extractTarget;
+}
+
+async function copyDebugProxyAssets() {
+    const version = process.argv[3];
```

Comment:
Nit; extract the 3 to a const somewhere.

Reply:
I'd actually take in the version as a parameter to this function so you don't have random functions reading from the command line input

Reply:
This now reads the debug proxy version from the `package.json`.